src/feature/flow_chart_panel/components/node-edit-menu/ParamField.tsx

```diff
@@ -60,6 +60,13 @@ const ParamField = ({
       );
     case "select":
       return <Select onChange={handleChange} data={options!} value={value} />;
+    case "node_reference":
```

Comment:
Node reference shouldn't be a simple text input rather it should be a dropdown of ID's of all nodes in the flowchart IMO

Reply:
Agreed, but it would take sometime for me to figure that out. Right now any node with references doesn't work at all so this is more like a placeholder. I plan to work on it after I finish my current task